.github/actions/setup-javascript/action.yml

```diff
@@ -0,0 +1,15 @@
+# currently not used because `uses` is not supported within composite steps
+# https://github.com/actions/runner/pull/1040
```

Comment:
This and setup-project aren't actually being used now, because you can't use `uses` within a composite step. I left it in because hopefully in the near future it will be supported.